src/css/view/woo-product-list.scss

```diff
@@ -767,4 +767,15 @@ $spacer-lg: $spacing5;
     ins, mark {
         background: unset;
     }
+}
+
+.theme-blocksy {
+    .eael-product-list-add-to-cart-button {
+        .button:before {
+            width: fit-content;
+            filter: none !important;
+            opacity: 1;
+            z-index: 0;
```

Comment:
Can we use `z-index: 1`?

Reply:
> Can we use `z-index: 1`?

@himadree bhai, how about this one? Should we update the z-index?

Reply:
Bhai z-index should "0", to work properly.